setup.py

```diff
@@ -42,7 +42,8 @@ def after_install():
         'mypy',
         'alembic',
         'configparser',
-        'pygithub'
+        'pygithub',
+        'googletrans==4.0.0-rc1'
```

Comment:
Is there a really specific reason behind this specific version?

Reply:
Yes, if I don't specify this version, their is a bug (newest version from what I found on google). That's why i need this specific version.

Reply:
Okay perfect then!